aiohttp_jwt/middleware.py

```diff
@@ -79,6 +71,17 @@ def JWTMiddleware(
                             algorithms=algorithms,
                         )
                         request[request_property] = decoded
+
```

Comment:
Maybe it would be better to leave only the expression that raises an exception in the try-except. 
https://github.com/hzlmn/aiohttp-jwt/blob/cd8a3882fe19eac08ffcfc03ee79b8ae9064dc55/aiohttp_jwt/middleware.py#L66-L87